hooks/boot/05-remove_hostname_check.rb

```diff
@@ -0,0 +1,7 @@
+# remove hostname check if it exists
+check_file = File.join(KafoConfigure.root_dir, 'checks', 'hostname.rb')
+begin
+  File.delete(check_file)
+  logger.info "The check #{check_file} was removed."
+rescue Exception
+end
```

Comment:
We'll disable katello-installer hostname checks because they come before we've had a chance to do any configuration. The provisioning wizard will provide validation instead.